openedx/core/lib/tests/test_derived.py

```diff
@@ -22,7 +22,9 @@ def setUp(self):
         derived('DERIVED_VALUE', 'ANOTHER_DERIVED_VALUE')
         self.module.DICT_VALUE = {}
         self.module.DICT_VALUE['test_key'] = lambda settings: settings.DERIVED_VALUE * 3
-        derived(('DICT_VALUE', 'test_key'))
+        derived_collection_entry('DICT_VALUE', 'test_key')
+        self.module.DICT_VALUE['list_key'] = ['first', lambda settings: settings.DERIVED_VALUE]
```

Comment:
Nit: `first` -> `zeroeth`?

Reply:
I think I'll just change it to something like "not derived".